apiserver/common/modelmanagerinterface.go

```diff
@@ -215,3 +215,26 @@ func (st modelManagerStateShim) AllVolumes() ([]state.Volume, error) {
 	}
 	return model.AllVolumes()
 }
+
+// ModelConfig returns the underlying model's config. Exposed here to satisfy the
+// ModelBackend interface.
+func (st modelManagerStateShim) ModelConfig() (*config.Config, error) {
+	model, err := st.State.Model()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	return model.ModelConfig()
+}
+
+// [TODO: Eric Claude Jones] This method ignores an error for the purpose of
+// expediting refactoring for CAAS features (we are avoiding changing method
+// signatures so that refactoring doesn't spiral out of control). This method
+// should be deleted immediately upon the removal of the ModelTag method from
+// state.State.
+func (st modelManagerStateShim) ModelTag() names.ModelTag {
```

Comment:
Remove this method - in the places which called ModelTag(), we can either use names.NewModelTag(st.ModelUUID()) or just use st.ModelUUID() directly. We can remove ModelTag() from the ModelManageBackend interface